eWonConnector-gateway/src/main/java/com/hms_networks/americas/sc/ignition/comm/responses/Talk2MResponse.java

```diff
@@ -0,0 +1,58 @@
+package com.hms_networks.americas.sc.ignition.comm.responses;
+
+import com.google.gson.Gson;
+
+import java.time.Instant;
+import java.util.Date;
+
+/**
+ * An abstract class for responses from a Talk2M API. This class contains any common fields and/or
+ * functionality for all Talk2M responses.
+ *
+ * @author HMS Networks, MU Americas Solution Center
+ * @since 2.0.0
+ * @version 1.0.0
+ */
+public class Talk2MResponse {
+
+  /**
+   * The success status of the logout request.
+   *
+   * @since 1.0.0
+   */
+  private boolean success;
+
+  /**
+   * Gets the success status of the login request.
+   *
+   * @return The success status of the login request.
+   * @since 1.0.0
+   */
+  public boolean getSuccess() {
+    return success;
+  }
+
+  /**
+   * Gets an instance of the specified subclass of {@link Talk2MResponse} from a JSON string.
+   *
+   * @param json The JSON string to parse.
+   * @return An instance of the specified subclass of {@link Talk2MResponse} parsed from the JSON
+   *     string.
+   * @since 1.0.0
+   */
+  public static <T extends Talk2MResponse> T getFromJson(String json, Class<T> clazz) {
+    return new Gson().fromJson(json, clazz);
```

Comment:
try/catch JsonSyntaxException? or perhaps add throws to method

Reply:
Changed